models/git/branch.go

```diff
@@ -382,17 +382,21 @@ func RenameBranch(ctx context.Context, repo *repo_model.Repository, from, to str
 }
 
 // FindRecentlyPushedNewBranches return at most 2 new branches pushed by the user in 6 hours which has no opened PRs created
-func FindRecentlyPushedNewBranches(ctx context.Context, repoID, userID int64) (BranchList, error) {
+// except the indicate branch
+func FindRecentlyPushedNewBranches(ctx context.Context, repoID, userID int64, excludeBranchName string) (BranchList, error) {
 	branches := make(BranchList, 0, 2)
 	subQuery := builder.Select("head_branch").From("pull_request").
 		InnerJoin("issue", "issue.id = pull_request.issue_id").
 		Where(builder.Eq{
 			"pull_request.head_repo_id": repoID,
 			"issue.is_closed":           false,
 		})
-	err := db.GetEngine(ctx).
-		Where("pusher_id=? AND is_deleted=?", userID, false).
-		And("updated_unix >= ?", time.Now().Add(-time.Hour*6).Unix()).
+	sess := db.GetEngine(ctx).
+		Where("pusher_id=? AND is_deleted=?", userID, false)
+	if excludeBranchName != "" {
```

Comment:
In which case the excludeBranchName can be empty string?

Even if `excludeBranchName == ""`, the expression `name <> ''` is still right for the query.

Reply:
Done.